samples/nrf9160/aws_fota/README.rst

```diff
@@ -29,6 +31,57 @@ Building and running
 
 .. include:: /includes/build_and_run_nrf9160.txt
 
+Prerequisites
+*************
+
+The connected device must have the same MQTT Client ID as the thing name created on AWS IoT.
+It also needs a policy attached to the certificates, and the certificates need to be active.
+
+Below is a very permissive policy which should be used for **testing only** which you can attach to the certificates if you are unsure on what policies are needed for your device. To create a policy go to `AWS IoT Console
+<https://console.aws.amazon.com/iot/home>`_ Secure>Policies>Create.
+
+.. code-block:: javascript
+
+    {
+      "Version": "2012-10-17",
+      "Statement": [
+          {
+            "Effect": "Allow",
+            "Action": "iot:*",
+            "Resource": "*"
+          }
+       ]
+     }
+
+To create a thing log in to the `AWS IoT Console
+<https://console.aws.amazon.com/iot/home>`_ Manage>Things>Create>Create a single thing.
+Give it a name which corresponds to your MQTT Client ID.
+Chose next either add your certificates or chose to create certificates, press activates and remember to download one of the `root CA certificates
+<https://docs.aws.amazon.com/iot/latest/developerguide/managing-device-certs.html#server-authentication>`_ and the generated certificates.
+
+Afterward, attach the policy you created before and put the information from the Root CA, Public Certificate, and Private Key into `certificates.h`.
+After this is done deselect the option in menuconfig which is described as *Use provisioned certificates*.
+This will write your new certificates to the secure tag defined in menuconfig into the modem.
+
+
+Usage
+*****
+
+Flash the example onto the board with the correct AWS IoT broker configured.
+If you haven't written certificates to a secure tag in the modem, use the provided ``certificates.h`` file to write your AWS IoT MQTT broker certificates into the modem.
+This only has to be done once so once you have configured it you can disable this option for the next builds.
+When the example has successfully connected to the AWS IoT broker you should log in to your `AWS IoT Console
+<https://console.aws.amazon.com/iot/home>`_ then go to *manage>jobs>create* chose **Create custom job** give a unique *Job ID* select the *device to update* and add a *job file*.
+Chose *snapshot* as job type and press next.
+
+Additional notes:
+The device can be created through AWS IoT things and should be associated with the certificates flashed onto the device.
```

Comment:
What does this sentence mean? What is a "device" in this case? And does "should" mean the user should do something, or that it should happen automatically?

Reply:
I guess "thing" would be the correct word.

This is something the user should do, you'll be prompted to create certificates if you create a thing.

Reply:
Ok. I'd say try to put all explanations about certificates in one place, if possible :). I realize they are quite an integral part.

Reply:
Yup, without it. It won't work I'll see if I can find a way to put it all in one place.

Reply:
Tried to collect the information into the prerequisite section does it make more sense now?